examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    )
</pre><p>can we add more sequential and non-sequential features?</p><p> </p><p>like</p><p> </p><p><code>"device_class": ["first"],</code></p><p>"hotel_country": ["list"],</p><p>&nbsp;&nbsp;"weekday_checkin": ["list"],</p><p>&nbsp;"weekday_checkout": ["list"],</p><p> </p><p>you can generate temporal features like below but note that <code>checkin  should be datetime object first.</code></p><p> </p><pre class="ql-syntax" spellcheck="false">
weekday_checkin = (
&nbsp; &nbsp; ["checkin"]
&nbsp; &nbsp; &gt;&gt; nvt.ops.LambdaOp(lambda col: col.dt.weekday)
&nbsp; &nbsp; &gt;&gt; nvt.ops.Rename(name="weekday_checkin")
)


weekday_checkout = (
&nbsp; &nbsp; ["checkout"]
&nbsp; &nbsp; &gt;&gt; nvt.ops.LambdaOp(lambda col: col.dt.weekday)
&nbsp; &nbsp; &gt;&gt; nvt.ops.Rename(name="weekday_checkout")
)


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>added additional features</p>

Reply:
<p>I think you need to add the <code>sort_cols="checkin" 
</code>arg to the <code>nvt.ops.Groupby</code> op , to ensure that each trip is sorted by check-in, as done in the Groupby op in this <a href="https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/examples/end-to-end-session-based/01-ETL-with-NVTabular.ipynb" target="_blank">example</a></p>

Reply:
Thank you very much for this @gabrielspmoreira! I've made the change